### ISSUE TITLE
Multi-gpu training

### ISSUE THREAD

**Issue by zhengzhezhe** [Plachtaa/seed-vc#101]
Hello, I want to use two or four more Gpus to finetune the model to reduce the training time of the model, may I ask how should I modify the code，or do you have a script for multi-gpus training?

**Comment by Plachtaa** [comment]
single gpu training script is clear enough
for multi-gpu training, someone knows how to do it if he has the will ✋

**Comment by ShakeYoung** [comment]
> Hello, I want to use two or four more Gpus to finetune the model to reduce the training time of the model, may I ask how should I modify the code，or do you have a script for multi-gpus training?

Do you have mutil GPU training code now? I also need to conduct multi GPU training here. Currently, I have partially rewritten the code, but the trained model has significant additional noise in the inference results. I don't know what the reason is. If you have training code for multiple GPUs, could you share it so we can discuss it together? Thank you 🙏
